scripts/release/mule/sign/sign.sh

```diff
@@ -56,8 +64,12 @@ shasum -a 512 *.tar.gz *.deb *.rpm >> "$HASHFILE"
 gpg -u "$SIGNING_KEY_ADDR" --detach-sign "$HASHFILE"
 gpg -u "$SIGNING_KEY_ADDR" --clearsign "$HASHFILE"
 
-gpg -u "$SIGNING_KEY_ADDR" --clearsign "$STATUSFILE"
-gzip -c "$STATUSFILE.asc" > "$STATUSFILE.asc.gz"
+# The status file *should* be present, but let's check to be safe.
+if [ -f "$STATUSFILE" ]
+then
+    gpg -u "$SIGNING_KEY_ADDR" --clearsign "$STATUSFILE"
+    gzip -c "$STATUSFILE.asc" > "$STATUSFILE.asc.gz"
+fi
```

Comment:
Is it considered an error if the status file does not exist?  Should this print an error message and return an error code if it is missing?

Reply:
I think there's a separate bug where the status file is getting put in the wrong place. The file is not required and not published, so I think this is a workaround; alternatively we could fix the placement of the status file and nip this in the bud.

Reply:
Yes, optimally the build status file is put back in its original location.

Reply:
I'm going to remove that conditional.